srcpkgs/polybar/template

```diff
@@ -51,6 +51,6 @@ if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 fi
 
 post_install() {
-	rm -rf ${DESTDIR}/usr/share/doc/polybar
+	find ${DESTDIR}/usr/share/doc/polybar/ -mindepth 1 ! -name config -delete
```

Comment:
IMO, it's better to remove everything is `/usr/share/doc/polybar` and `vdoc config` or `vsconf config`.

Reply:
The only problem with this approach is that it breaks if upstream changes where they put the example config. That isn't the case here.

Reply:
OK, I've modifed it to `rm` the directory and then `vdoc` the `config` file.  Here is the output of `xls`:

```
/usr/bin/polybar
/usr/bin/polybar-msg
/usr/share/bash-completion/completions/polybar
/usr/share/doc/polybar/config
/usr/share/licenses/polybar/LICENSE
/usr/share/man/man1/polybar.1
/usr/share/man/man5/polybar.5
/usr/share/zsh/site-functions/_polybar
/usr/share/zsh/site-functions/_polybar_msg
```